Archive/Wiedijk100Theorems/HeronsFormula.lean

```diff
@@ -12,6 +12,10 @@ This file proves Theorem 57 from the [100 Theorems List](https://www.cs.ru.nl/~f
 also known as Heron's formula, which gives the area of a triangle based only on its three sides'
 lengths.
 
+## TODO
+
+Change variable names here: https://leanprover-community.github.io/100.html#57
+
```

Comment:
Since the 100-theorems and 1000-theorems pages are updated automatically when changes are made to mathlib, none of these TODOs are appropriate (the automation updates statements on those pages including variable names, but it doesn't know how to remove a TODO in mathlib). With all these new TODOs removed again:

maintainer delegate

Reply:
Great! Thanks for the info! I removed the TODOs.